### ISSUE TITLE
[V3 CustomCommands] Allow option to fetch Markdown version of the text

### ISSUE THREAD

**Issue by smlbiobot** [Cog-Creators/Red-DiscordBot#820]
Unless I have mistaken, if the text I use in the custom commands contains markdown for formatting, there is currently no way to copy the full markdown from the app for easy edits. Right now I have to re-create the markdown every time because I can only get plain text if copied from the app.

This is not an issue if it was the same person who created the original text, but it became a hassle whenever I need to edit someone else’s text. In some servers that I admin, the customcoms are rather long so I would like an option for me to copy the markdown source, perhaps placed inside a code block.

Please let me know if that option is already there and I just didn’t know about it. And if not, it would be awesome if you can implement that feature.

Thanks!

**Comment by skmendez** [comment]
I have a PR that would add this, but the problem is that there is no way in Discord to show 3 contiguous backticks inside of a codeblock. In normal Markdown, we can display backticks as such:

```````
We use 5 backticks:
`````
So we can use 3 backticks inside the codeblock:
```
Look at em!
```
And now we exit the code block
`````
Isn't that magical!
```````
I also used 7 backticks to escape that codeblock. Discord, unfortunately, does not support this.

**Comment by Tobotimus** [comment]
As stated on #1352 I'd happily accept this as an addition to V3 :)

**Comment by Tobotimus** [comment]
Resolved by #2204.
